src/scaffoldmaker/meshtypes/meshtype_3d_bladder1.py

```diff
@@ -353,25 +353,19 @@ def getDefaultOptions(cls, parameterSetName='Default'):
         }
         if 'Cat 1' in parameterSetName:
             options['Wall thickness'] = 0.007  # was 1.5 * 2.0 / 178.08271473110773
-            options['Ureter position around'] = 0.67  # should be on the dorsal part (> 0.5)
         if 'Human 1' in parameterSetName:
             options['Wall thickness'] = 0.043  # was 3.0 * 2.0 / 327.63
-            options['Ureter position around'] = 0.67  # should be on the dorsal part (> 0.5)
         if 'Mouse 1' in parameterSetName:
             options['Wall thickness'] = 0.008  # was 0.5 * 2.0 / 72.004931756029
-            options['Ureter position around'] = 0.67  # should be on the dorsal part (> 0.5)
         if 'Pig 1' in parameterSetName:
             options['Wall thickness'] = 0.016  # was 2.5 * 2.0 / 505.2045520616655
-            options['Ureter position around'] = 0.67  # should be on the dorsal part (> 0.5)
         if 'Rat 1' in parameterSetName:
             options['Wall thickness'] = 0.006  # was 0.3 * 2.0 / 110.40996641878101
-            options['Ureter position around'] = 0.67  # should be on the dorsal part (> 0.5)
         if 'Material' in parameterSetName:
             options['Number of elements along dome'] = 8
             options['Number of elements along neck'] = 4
             options['Number of elements around'] = 8
             options['Wall thickness'] = 0.016  # an average of the wall thicknesses for above species
```

Comment:
Make the wall thickness in material coordinates 0.02 so it's a nice round number (and closer to half the human wall thickness). I'm still surprised by the variations here...
```
            options['Wall thickness'] = 0.02  # an average of the wall thicknesses for above species, rounded
```

Reply:
Done.